src/Query/Builder.php

```diff
@@ -538,9 +596,13 @@ public function generateCacheKey()
         return md5(serialize(array_values($key)));
     }
 
-    /** @inheritdoc */
-    public function aggregate($function, $columns = [])
+    /** @return ($function is null ? AggregationBuilder : mixed) */
+    public function aggregate($function = null, $columns = [])
     {
+        if ($function === null) {
+            return $this->getAggregationBuilder();
```

Comment:
I understand this ignores the `$columns` parameter, but would it be worth asserting that `$columns` is still empty and that a user didn't specify some value(s) there?

Reply:
Good idea. Done.